apps/civicsignalblog/src/payload/access/canAccessApplication.js

```diff
@@ -0,0 +1,10 @@
+export default function canAccessApplication(user, searchString) {
+  if (user) {
+    const app = user.currentApp || user.defaultApp;
+    if (app) {
+      return app === searchString.toLowerCase();
+    }
+    return false;
+  }
+  return false;
```

Comment:
```suggestion
  if (user) {
    const app = user.currentApp || user.defaultApp;
    return app === searchString.toLowerCase();
  }
  return false;
```

Don't think we need `if`/multiple `return`s